drf_extra_fields/fields.py

```diff
@@ -177,6 +188,27 @@ class RangeField(DictField):
         'bound_ordering': _('The start of the range must not exceed the end of the range.'),
     })
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        checks = self.check()
+
+        for check in checks:
+            logger.error(check)
+
+    def check(self, **kwargs):
+        """Check if postgres is none. you can also extend this function to check more things.
+        """
+        errors = []
+        if postgres_fields is None:
+            errors.append(
+                checks.Error(
+                    "'psgl2' is required to use {name}".format(name=self.__class__.__name__),
+                    hint="Install the 'psycopg2' library from 'pip'",
+                    obj=self
+                )
+            )
+        return errors
```

Comment:
Is this working as expected? It doesn't seem to me that you are using checks framework as expected.

Please see how it's used: https://docs.djangoproject.com/en/3.1/topics/checks/#writing-your-own-checks

You need to register the check method. And check method should be something like `def example_check(app_configs, **kwargs):`

Not sure if we can benefit from checks framework at this point. 

How about we put something like below in the init method?
```
if postgres_fields is None:
    assert False, "'psgl2' is required to use {name}. Please install the  'psycopg2' library from 'pip'".format(name=self.__class__.__name__)
```
I think this is simple enough and gets the job done.